lib/style/module_directives.ex

```diff
@@ -76,36 +76,53 @@ defmodule Styler.Style.ModuleDirectives do
 
   def run({{:defmodule, _, children}, _} = zipper, ctx) do
     [{:__aliases__, _, aliases}, [{{:__block__, do_meta, [:do]}, _module_body}]] = children
-    # Move the zipper's focus to the module's body
-    name = aliases |> List.last() |> to_string()
-    add_moduledoc? = do_meta[:format] != :keyword and not String.ends_with?(name, @dont_moduledoc)
-    body_zipper = zipper |> Zipper.down() |> Zipper.right() |> Zipper.down() |> Zipper.down() |> Zipper.right()
-
-    case Zipper.node(body_zipper) do
-      {:__block__, _, _} ->
-        {:skip, organize_directives(body_zipper, add_moduledoc?), ctx}
-
-      {:@, _, [{:moduledoc, _, _}]} ->
-        # a module whose only child is a moduledoc. nothing to do here!
-        # seems weird at first blush but lots of projects/libraries do this with their root namespace module
-        {:skip, zipper, ctx}
-
-      only_child ->
-        # There's only one child, and it's not a moduledoc. Conditionally add a moduledoc, then style the only_child
-        if add_moduledoc? do
-          body_zipper
-          |> Zipper.replace({:__block__, [], [@moduledoc_false, only_child]})
-          |> Zipper.down()
-          |> Zipper.right()
-          |> run(ctx)
-        else
-          run(body_zipper, ctx)
-        end
+
+    if do_meta[:format] == :keyword do
+      {:skip, zipper, ctx}
+    else
```

Comment:
skip keyword defmodules. everything below this is just indentation, nothing else in this function head changed